pdfminer/layout.py

```diff
@@ -281,7 +281,7 @@ class LTChar(LTComponent, LTText):
     """Actual letter in the text as a Unicode string."""
 
     def __init__(self, matrix, font, fontsize, scaling, rise,
-                 text, textwidth, textdisp, ncs, graphicstate):
+                 text, textwidth, textdisp, ncs, graphicstate, glyph_bbox):
```

Comment:
Does this need a default argument by any chance?

Reply:
Nope.